Gamma_Reference.py

```diff
@@ -1,41 +1,205 @@
+import numpy as np
+import peakutils
+import pandas as pd
+import shutil as sh
+import fileinput
+import SPEFile
+import os
 
+energy_list = [351.93, 583.19, 609.31, 911.20, 1460.82, 1764.49,
+               2614.51]
+cal_headers = [351.93, 583.19, 609.31, 911.20, 1460.82, 1764.49,
+               2614.51, 'Output']
 
-class Reference(object):
+def acquire_files():
```

Comment:
I'm not sure what happened here, but it looks like `Gamma_Reference.py` now contains the contents of `calibrate.py`.